opm/input/eclipse/Schedule/Schedule.hpp

```diff
@@ -20,22 +20,21 @@
 #define SCHEDULE_HPP
 
 #include <cstddef>
+#include <ctime>
 #include <map>
 #include <memory>
 #include <optional>
-#include <ostream>
+#include <iosfwd>
 #include <string>
 #include <unordered_map>
-#include <unordered_set>
 #include <utility>
 #include <vector>
 
-#include <time.h>
```

Comment:
This file uses bare `time_t`s so we need one of the following

* `#include <time.h>` as we currently do
* Add a `using std::time_t` before the first reference to an undecorated `time_t` type.
* Replace all current `time_t`s with `std::time_t` instead.

Reply:
You are aware that we include ctime which seems to be the correct header include in C++ code. I agree with std::time_t for good style but intend to merge this without as it will work with our supported compilers.

Reply:
> You are aware that we include `<ctime>` which seems to be the correct header include in C++ code.

Yes.  On the other hand the C++ `<ctime>` header declares `std::time_t` and does not declare the undecorated name `time_t`.  If we're going to continue to use the undecorated name then we either need to continue to include `<time.h>`&mdash;which does define that name&mdash;or we'll have to insert `using std::time_t` somewhere early in `Schedule.hpp`.  Doing anything else is not portable and ties us to a specific version of a specific compiler.

> `std::time_t` for good style but intend to merge this without as it will work with our supported compilers.

It's about correctness, not style.  There's absolutely no reason to expect
```C++
#include <ctime>

time_t f();
```
to work correctly/at all in a future version of our supported compilers.

Reply:
Yes i will address this oversight of mine (using decoration), just did not get around to it today.

Reply:
interestingly one of the methods was declared time_t and implemented as std::time_t. in any case they are now all std::time_t.